src/data.json

```diff
@@ -0,0 +1,18 @@
+{
+  "fontTitle": "Maname fontpad",
+  "fontMeta1": "v0.001-alpha",
+  "fontMeta2": "Updated on 27 Dec 2016",
+  "settings":
+  [
+ {"i": "10","fontSize": 40,"lineHeight": 1.5,"letterSpacing": 0, "content": "The Screen in Review Elephant Walk opens at Astor theatre"},
```

Comment:
`i` means the `id` right? Better use the full name otherwise the user might get confused.